RELEASENOTES-1.2.docu

```diff
@@ -163,4 +163,9 @@
   <build-id value="6215"><add-note> Improve porting rules for method
       return values: In some simple cases, avoid storing the method's
       return value in an intermediate local variable.</add-note></build-id>
+  <build-id value="next"><add-note> If a bank declared with
+      <tt>use_io_memory=false</tt> in a DML 1.4 file is imported from
+      a DML 1.2 device, then the device will now honor overrides of
+      the <tt>transaction_access</tt> method. This simplifies future
```

Comment:
Wait, maybe I lost a piece of the thread along the line -- isn't this only true if `dml12_compat_io_memory_access` is instantiated?

Reply:
Or if transaction_access is called. Should indeed be rephrased, it was written for an earlier revision of the PR.

Reply:
Rephrased.